utils/starlark.js

```diff
@@ -104,7 +104,7 @@ const sortCallArgItems /*: SortCallArgItems */ = (code, caller, argName) => {
           .filter(Boolean)
           .sort()
           .join(',');
-        return `[${sorted},${dedent}]`.replace(/,]/, ']');
+        return `[${sorted},${dedent}]`.replace(/,]/, ']').replace(/\[,/, '[');
```

Comment:
There was a bug in this util that was generating a dangling comma in the deps list for BUILD.bazel files which looked like this:

```
web_library(
    name = "library",
    deps = [,
        
    ],
    srcs = glob(["**"], exclude = ["dist/**"]),
)
```

This was necessary to fix the `testBatchTestGroup` test